### PR TITLE
Command handler cleanup

### PR DESCRIPTION
**Pull request by saramaebee**

### CODE REVIEW

File: app.js [hunk 1]
```diff
@@ -2,7 +2,7 @@
 const config = require("./config.json");
 
 // Package Dependencies
-const fileSystem = require("fs");
+const filesystem = require("fs");
```
Comment (Bas-Sarrechia) [inline app.js:5]: fileSystem was correct -> camelcasing. file + system (two different words)

File: app.js [hunk 2]
```diff
@@ -26,27 +26,22 @@ client.on("ready", () => {
 // Run when a message is sent
 client.on("message", (message) => {
 	for (const commandFile in commands) {
+		// Ensure the commands object has a key of the commandFile value
+		if (!commands.hasOwnProperty(commandFile)) {
+			continue;
+		}
```
Comment (Bas-Sarrechia) [inline app.js:32]: this part can likely be written better and is likely flawed in logic. try to reconsider this continue; statement

File: app.js [hunk 3]
```diff
@@ -56,11 +51,15 @@ client.on("message", (message) => {
 
 		embed.setTitle("Commands");
 		for (const commandFile in commands) {
+			// Ensure the commands object has a key of the commandFile value
+			if (!commands.hasOwnProperty(commandFile)) {
+				continue;
+			}
```
Comment (Bas-Sarrechia) [inline app.js:57]: again see above comment.
-> reason potential breaking in future
-> excess of logic = loss in performance

**Review by Bas-Sarrechia — CHANGES_REQUESTED** [review]
preferable integrate tests with jester to confirm logic works.
See inline comments on todo.

File: app.js (hunk 2 above)
Comment (LamboCreeper) [reply app.js:32]: I'd rather `continue;` wasn't used.

**Comment by LamboCreeper** [comment]
Closed for reasons discussed within Discord.

**Pull request closed without merging**